packages/terra-section-header/src/SectionHeader.module.scss

```diff
@@ -27,19 +29,19 @@
       &:hover {
         background: var(--terra-section-header-hover-background, #d7d8d9);
         border-bottom: var(--terra-section-header-hover-border-bottom, 1px solid #c8cacb);
-        border-left: var(--terra-section-header-hover-border-left);
-        border-right: var(--terra-section-header-hover-border-right);
-        border-top: var(--terra-section-header-hover-border-top);
-        box-shadow: var(--terra-section-header-hover-box-shadow);
+        border-left: var(--terra-section-header-hover-border-left), 0 none;
+        border-right: var(--terra-section-header-hover-border-right, 0 none);
+        border-top: var(--terra-section-header-hover-border-top, 0 none);
+        box-shadow: var(--terra-section-header-hover-box-shadow, none);
         color: var(--terra-section-header-hover-color, #1c1f21);
       }
 
       &:focus {
         background: var(--terra-section-header-focus-background, #e8e9ea);
         border-bottom: var(--terra-section-header-focus-border-bottom, 1px solid #c8cacb);
-        border-left: var(--terra-section-header-focus-border-left);
-        border-right: var(--terra-section-header-focus-border-right);
-        border-top: var(--terra-section-header-focus-border-top);
+        border-left: var(--terra-section-header-focus-border-left, 0 none);
+        border-right: var(--terra-section-header-focus-border-right, 0 none);
+        border-top: var(--terra-section-header-focus-border-top, none);
```

Comment:
Should this be `0 none`?

Reply:
Fixed: https://github.com/cerner/terra-core/pull/3631/commits/e973191b95edd3ab18ac6ba51c0cef8336a4c536